services/QuillLMS/client/app/bundles/Staff/components/evidence/configureSettings/activityForm.tsx

```diff
@@ -253,6 +297,7 @@ const ActivityForm = ({ activity, handleClickArchiveActivity, requestErrors, sub
           handleSetPrompt={handleSetPrompt}
         />
       </form>
+      {invalid_highlights && invalid_highlights.length > 0 && renderInvalidHighlightLinks(invalid_highlights)}
```

Comment:
I think it would be nice to have this bool check above the return as a variable, something like `const invalidHighlightsPresent = invalid_highlights && invalid_highlights.length > 0`